documentation/src/main/asciidoc/topics/ref_rest_caches.adoc

```diff
@@ -656,3 +656,30 @@ POST /rest/v2/caches/{cacheName}?action=set-availability&availability={AVAILABIL
 
 [NOTE]
 You can set the availability of internal caches but this is subject to change in future {brandname} versions.
+
+[id='rest_v2_caches_set_topology_stable']
+= Set a Stable Topology
+By default, after a cluster shutdown, {brandname} waits for all nodes to join the cluster and restore the topology.
+However, it is possible to define the current cluster topology as stable for a specific cache using the REST operation.
+
+[source,options="nowrap",subs=attributes+]
+----
+POST /rest/v2/caches/{cacheName}?action=initialize&force={FORCE}
+----
+
+.Request Parameters
+
+|===
+|Parameter |Required or Optional |Value
+
+|`force`
+|OPTIONAL
+|true or false.
+|===
+
+* `force` is required when the number of missing nodes in the current topology is greater or equal to the number of owners.
+
+[IMPORTANT]
+====
+Manually installing a topology is unsafe as it can lead to data loss.
```

Comment:
Is this too string as there's scenarios where this action is unavoidable, hence why we provide the capability. How about something like:

"Manually installing a topology can lead to data loss, only perform this operation if the initial topology cannot be recreated"

Reply:
Updated.